plugins/search-backend-module-techdocs/src/collators/DefaultTechDocsCollatorFactory.ts

```diff
@@ -104,7 +104,18 @@ export class DefaultTechDocsCollatorFactory implements DocumentCollatorFactory {
       config.getOptionalBoolean(
         'techdocs.legacyUseCaseSensitiveTripletPaths',
       ) || false;
-    return new DefaultTechDocsCollatorFactory({ ...options, legacyPathCasing });
+    const locationTemplate = config.getOptionalString(
+      'search.techdocs.collators.locationTemplate',
```

Comment:
Yeah definitely feeling that `search.collators.techdocs.locationTemplate` would make more sense? 😅